contract/package.json

```diff
@@ -5,17 +5,6 @@
   "description": "Agoric Orchestration Contract",
   "type": "module",
   "scripts": {
-    "start:docker": "docker compose up -d",
-    "docker:logs": "docker compose logs --tail 200 -f",
-    "docker:bash": "docker compose exec agd bash",
```

Comment:
Since this `start:docker` and `docker:logs` are our getting started docs, I suggest that `yarn start:docker` and the like should give a diagnostic about what to do instead. (Be sure to give a failing exit code.)

Or... if it's easy enough to just make `docker:logs` work (i.e. show the agd logs) then we should do that. (likewise `docker:bash`)

Reply:
Good point.
- `start:docker` now points to readme in `multichain-testing` to setup env
- `docker:logs` now shows agd logs
- `docker:bash` now opens `agoriclocal-genesis-0` shell